cleaners/gimp_new.xml

```diff
@@ -22,6 +22,8 @@
 <cleaner id="gimp_new">
   <label>GIMP 2.10</label>
   <description>Graphics editor</description>
+  <running type="exe" os="windows">gimp-2.10.exe</running>
+  <running type="exe" os="linux">gimp-2.10</running>
```

Comment:
That works when GIMP is started from GNOME. If could also be started from shell as `gimp` and then detection will fail.
```
|   \_ gimp
|       \_ /usr/lib64/gimp/2.0/plug-ins/script-fu/script-fu -gimp 10 9 -run 0
```

Reply:
Hello,
I just tested on ubuntu as you can see on my screenshot and also on ArchLinux KDE by opening GIMP with the terminal
![Capture d’écran de 2021-12-29 19-15-28](https://user-images.githubusercontent.com/48923336/147692296-44317747-6750-4f8e-b2ad-4211e2c26fb5.png)